### ISSUE TITLE
Patterns don't match the files on Windows

### ISSUE THREAD

**Issue by falsandtru** [karma-runner/karma#3795]
Patterns match the files if the karma is started by gulp but don't match the files if started by npx or node_modules on Windows 10.

```js
module.exports = function (config) {
  config.set({
    basePath: '',
    frameworks: ['mocha'],
    browsers: ['Chrome'],
    singleRun: true,
  });
};
```

```console
$ npx karma start
$ node_modules/.bin/karma start
```

```js
  minimatch: Minimatch {
    options: {
      cwd: '/',
      follow: true,
      nodir: true,
      sync: true,
      nonegate: true,
      nocomment: true,
      allowWindowsEscape: true
    },
    set: [ [Array] ],
    pattern: '...\\node_modules\\karma-mocha\\lib\\adapter.js',
    regexp: null,
    negate: false,
    comment: false,
    empty: false,
    partial: false,
    globSet: [
      '...\\node_modules\\karma-mocha\\lib\\adapter.js'
    ],
    globParts: [ [Array] ]
  },
  options: {
    cwd: '/',
    follow: true,
    nodir: true,
    sync: true,
    nonegate: true,
    nocomment: true,
    allowWindowsEscape: true
  },
  matches: [ [Object: null prototype] {} ],
  found: []
```

```console
$ gulp karma
```

```js
  minimatch: Minimatch {
    options: {
      cwd: '/',
      follow: true,
      nodir: true,
      sync: true,
      nonegate: true,
      nocomment: true
    },
    set: [ [Array] ],
    pattern: '.../node_modules/karma-mocha/lib/adapter.js',
    regexp: null,
    negate: false,
    comment: false,
    empty: false,
    globSet: [
      '.../node_modules/karma-mocha/lib/adapter.js'
    ],
    globParts: [ [Array] ]
  },
  options: {
    cwd: '/',
    follow: true,
    nodir: true,
    sync: true,
    nonegate: true,
    nocomment: true
  },
  matches: [
    [Object: null prototype] {
      '.../node_modules/karma-mocha/lib/adapter.js': true
    }
  ],
  found: [
    '.../node_modules/karma-mocha/lib/adapter.js'
  ]
```

https://github.com/karma-runner/karma/blob/v6.3.20/lib/file-list.js#L71

**Comment by falsandtru** [comment]
Ah, not correct reason. Both commands don't find the files after reinstalling the dependencies with npm@8 (from npm@6).

**Comment by falsandtru** [comment]
Also broken with npm@6. Maybe some dependencies are broken.

**Comment by falsandtru** [comment]
Work with glob@7.2.0! The root cause is probably https://github.com/isaacs/node-glob/issues/471.

**Comment by isaacs** [comment]
Should be fixed by glob@7.2.3.  If it isn't, please post a new issue at https://github.com/isaacs/node-glob/issues, ideally with a reproduction case.  Thanks!

**Comment by falsandtru** [comment]
Thanks.
